content/docs/_index.md

```diff
@@ -7,10 +7,12 @@ sidebar_multicard: true
 icon: docs
 ---
 
-In addition, you might find these links interesting:
+In addition to the documentation, we provide the following resources:
 
-- Check out our [Help](/help) section, which includes our [Operations Guide](/help/ops), our [FAQ](/help/faq), and our [glossary](/help/glossary).
+- [FAQ](/faq) section
+- [Operations Guide](/docs/ops)
```

Comment:
No need for this link, it's shown right above as part of the section-level page code generation.